lib/jsdom/virtual-console.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+var EventEmitter = require("events").EventEmitter;
+
+var vConsole = {};
+
+var emitters = {};
+var windows = [];
+
+// Returns an EventEmitter that emits on any console events
+vConsole.getVirtualConsole = function(window) {
+  var virtualConsole = new EventEmitter();
+
+  saveEmitterForWindow(virtualConsole, window);
+
+  virtualConsole.sendTo = function(anyConsole) {
+
+    if (!isValidConsole(anyConsole, window)) {
```

Comment:
No need for this test IMO. If it doesn't have any console methods they'll get errors later, closer to the source.